Cargo.toml

```diff
@@ -23,6 +23,7 @@ tempdir = { version = "0.3", optional = true }
 serde = "1.0"
 serde_json = "1.0"
 serde_derive = "1.0"
+rustc-test = { git = "https://github.com/messense/rustc-test.git", branch = "update", optional = true}
```

Comment:
What's in this crate? Could you add it to `crates.io`?

Reply:
It's a fork of [servo/rustc-test](https://github.com/servo/rustc-test) with updates from rustc. It should be upstreamed to servo/rustc-test.

Reply:
I have opened a PR in rustc-test here: https://github.com/servo/rustc-test/pull/8